src/main/java/com/aws/greengrass/deployment/DeploymentService.java

```diff
@@ -377,7 +377,7 @@ private void persistGroupToRootComponents(DeploymentDocument deploymentDocument)
                 Map<String, Object> pkgDetails = new HashMap<>();
                 pkgDetails.put(GROUP_TO_ROOT_COMPONENTS_VERSION_KEY, pkgConfig.getResolvedVersion());
                 pkgDetails.put(GROUP_TO_ROOT_COMPONENTS_GROUP_CONFIG_ARN,
-                        deploymentDocument.getDeploymentId());
+                        deploymentDocument.getConfigurationArn());
```

Comment:
isn't this going to break existing installations? They will have an incorrect mapping from the old behavior right?

Reply:
before this change deploymentDocument stored configurationArn inside deploymentId (and deploymentId was not stored). I updated deploymentDocument such that deploymentId is in deploymentId and configurationArn is in configurationArn. There is no change in behavior here.